src/orion/core/cli/db/dump.py

```diff
@@ -0,0 +1,126 @@
+#!/usr/bin/env python
+# pylint: disable=,protected-access
+"""
+Storage export tool
+===================
+
+Export database content into a file.
+
+"""
+import logging
+import os
+
+from orion.core.cli import base as cli
+from orion.core.io import experiment_builder
+from orion.core.io.database import DatabaseError
+from orion.core.io.database.pickleddb import PickledDB
+from orion.storage.base import setup_storage
+
+logger = logging.getLogger(__name__)
+logger.setLevel(logging.DEBUG)
+
+DESCRIPTION = "Export storage"
+
+COLLECTIONS = {"experiments", "algo", "benchmarks", "trials"}
+
+
+def add_subparser(parser):
+    """Add the subparser that needs to be used for this command"""
+    dump_parser = parser.add_parser("dump", help=DESCRIPTION, description=DESCRIPTION)
+
+    cli.get_basic_args_group(dump_parser)
+
+    dump_parser.add_argument(
```

Comment:
We should add a `--force` option and avoid overwritting the output file if it already exists and `--force` is not passed.